docs/source/contents/4_draft_mapping.rst

```diff
@@ -1662,6 +1662,407 @@ opaque:hostItem
+
+.. code-block:: xml
+
+    <location>
+        <physicalLocation>Pi Beta Phi Fraternity</physicalLocation>
+        <physicalLocation displayLabel="Address">1154 Town and Country Commons Drive, Town and Country, Missouri 63017</physicalLocation>
+        <shelfLocator>Box 36, Folder 14</shelfLocator>
+    </location>
+
+.. code-block:: turtle
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators> .
+    @prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+
+    <https://example.org/objects/1>
+        relators:rps "Pi Beta Phi Fraternity" ;
+        skos:note "Shelf locator: Box 36, Folder 14" .
+
+physicalLocation with @displayLabel="Collection"
+------------------------------------------------
+
+Use Case
+^^^^^^^^
+In a few of collections for Arrowmont, we will find items having a physicalLocation subelement with the displayLabel attribute of Collection, along with Detailed Location, City and State.
+
+Justification
+^^^^^^^^^^^^^
+Because these records do not already have a dbo:collection predicate, we will transcribe the string literal to dbo:collection for `physicalLocation[@displayLabel=Collection]`. No other data here needs to be retained and will be dropped.
+
+Xpath
+^^^^^
+
+`mods:location/mods:physicalLocation[@displayLabel=Collection]`
+
+Decision
+^^^^^^^^
+We will keep the string for the physicalLocation instance with displayLabel=Collection and transcribe this to literal for dbo:collection.
+
+Like when physicalLocation has no displayLabel, Repository is retained as relators:rps. All other displayLabel (Detailed Location, City, State) data is dropped.
+
+`Example record from volvoices:2199 <https://digital.lib.utk.edu/collections/islandora/object/volvoices:2199/datastream/MODS/view>`_
+
+.. code-block:: xml
+
+    <location>
+        <physicalLocation displayLabel="Collection">Archives Collection</physicalLocation>
+        <physicalLocation displayLabel="Repository">Arrowmont School of Arts and Crafts</physicalLocation>
+        <physicalLocation displayLabel="Detailed Location"/>
+        <physicalLocation displayLabel="City">Gatlinburg</physicalLocation>
+        <physicalLocation displayLabel="State">Tennessee</physicalLocation>
+    </location>
+
+.. code-block:: turtle
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators> .
+    @prefix dbo: <http://dbpedia.org/ontology/> .
+
+    <https://example.org/objects/1>
+        relators:rps <http://id.loc.gov/authorities/names/no2001080757> ;
+        dbo:collection "Archives Collection" .
+
+url
+---
+
+Use Case
+^^^^^^^^
+
+Some of our metadata for volvoices may contain self-referential URL locations. The data contained in these directly reference objects in our current Islandora 7 build and the object’s datastreams.
+
+Justification
+^^^^^^^^^^^^^
+
+This is self-referential and has no value in a new system.
+
+Xpath
+^^^^^
+
+`mods:location/mods:url`
+
+Decision
+^^^^^^^^
+This is dropped.
```

Comment:
We haven't worried too much about voice (we probably should), but I think we should switch to:

```
Drop this.
```

or

```
Don't migrate.
```